__init__.py

```diff
@@ -15,95 +15,406 @@
-        self.extended_delay = self.config.get('extended_delay')
-        self.file_path = join(dirname(__file__), self.config.get('filename'))
+        super(AlarmSkill, self).__init__()
+        self.time_format = self.config_core.get('time_format')
+        self.should_converse = False
+        self.stop_notify = False
+        self.allow_notify = False
+        self.converse_context = {}
 
     def initialize(self):
-        super(AlarmSkill, self).initialize()
-        intent = IntentBuilder(
-            'AlarmSkillStopIntent').require('AlarmSkillStopVerb') \
-            .require('AlarmSkillKeyword').build()
-        self.register_intent(intent, self.__handle_stop)
-
-    def load_data(self):
-        try:
-            with self.file_system.open(self.PENDING_TASK, 'r') as f:
-                self.data = yaml.safe_load(f)
-                assert self.data
-        except:
-            self.data = {}
-
-    def load_repeat_data(self):
-        try:
-            with self.file_system.open(self.REPEAT_TASK, 'r') as f:
-                self.repeat_data = yaml.safe_load(f)
-                assert self.repeat_data
-        except:
-            self.repeat_data = {}
-
-    def __handle_stop(self, message):
-        if self.alarm_on:
-            self.speak_dialog('alarm.off')
-        self.alarm_on = False
-
-    def notify(self, timestamp):
-        with self.LOCK:
-            if self.data.__contains__(timestamp):
-                volume = None
-                self.alarm_on = True
-                delay = self.__calculate_delay(self.max_delay)
-
-                while self.alarm_on and datetime.now() < delay:
-                    play_mp3(self.file_path).communicate()
-                    self.speak_dialog('alarm.stop')
-                    time.sleep(self.repeat_time + 2)
-                    if not volume and datetime.now() >= delay:
-                        mixer = Mixer()
-                        volume = mixer.getvolume()[0]
-                        mixer.setvolume(100)
-                        delay = self.__calculate_delay(self.extended_delay)
-                if volume:
-                    Mixer().setvolume(volume)
-                self.remove(timestamp)
-                self.alarm_on = False
-                self.save()
-
-    @staticmethod
-    def __calculate_delay(seconds):
-        return datetime.now() + timedelta(seconds=seconds)
-
-    def save(self):
-        with self.file_system.open(self.PENDING_TASK, 'w') as f:
-            yaml.safe_dump(self.data, f)
-        with self.file_system.open(self.REPEAT_TASK, 'w') as f:
-            yaml.safe_dump(self.repeat_data, f)
-        if not self.alarm_on:
-            self.schedule()
+        # self.register_intent_file('set.morning.intent', self.set_morning)
+        # self.register_intent_file('set.sunrise.intent', self.set_sunrise)
+        # self.register_intent_file('delete.all.intent', self.delete_all)
+        # self.register_entity_file('exceptdaytype.entity')
```

Comment:
Should we remove these and if these are to-be implemented, paste them into an issue?